strictdoc/backend/excel/import_/excel_to_sdoc_converter.py

```diff
@@ -231,7 +230,7 @@ def extend_requirement_with_extra_columns(
         return requirement
 
     @staticmethod
-    def parse(input_path, title=None):
+    def parse(input_path):
```

Comment:
@stanislaw, can this commit be reverted please? We use the title argument in our code in https://github.com/nmfta-repo/nmfta-vehicle_cybersecurity_requirements/pull/10

Reply:
Sure, could you do it? I generally try to avoid optional arguments but in this case, it would be great if you added the explanation that the intended use case if for the `.convert` method to be called from external scripts. Note that I have removed the `parse` method to simplify the code structure - now there is only `convert`.